### ISSUE TITLE
Translate methane into other languages

### ISSUE THREAD

**Issue by AnOpenSauceDev** [AnOpenSauceDev/Methane-mod#19]
^ pretty self-explanatory, not really a high priority right now.

**Comment by AnOpenSauceDev** [comment]
given the maturity of the mod, this seems to be a logical step forward.

**Comment by Cyclopropinon** [comment]
what languages are currently available?

**Comment by AnOpenSauceDev** [comment]
> what languages are currently available?

two variants of english, and an incomplete _deutsch_ (german) translation.

**Comment by Cyclopropinon** [comment]
i can help with german and chinese(simplified) (both native speaker)
tell me how

**Comment by AnOpenSauceDev** [comment]
> i can help with german and chinese(simplified) (both native speaker) tell me how

@Cyclopropinon 

Sure! it's basically like making a resourcepack.

go into `src/main/resources/assets/methane/lang`

all the files there are language files, and they are named after their corresponding country codes 
(i.e. `en_au` for australian english, `en_us` for american english, etc.)

to create a translation, create (or enter) the file, and label the translation key (stuff like the `methane.` or `text.autoconfig.` things). It should be pretty easy if you follow the pre-made english files as an example.

**Comment by Cyclopropinon** [comment]
oof
![grafik](https://user-images.githubusercontent.com/119750500/235998194-c89b11a9-10ec-44e4-b90c-cfb5068fe827.png)
the "ausgabefenster" is empty

**Comment by Cyclopropinon** [comment]
found the issue
`Remote: Permission to AnOpenSauceDev/Methane-mod.git denied to Cyclopropinon.`
@AnOpenSauceDev pls gimme perms

**Comment by AnOpenSauceDev** [comment]
> found the issue `Remote: Permission to AnOpenSauceDev/Methane-mod.git denied to Cyclopropinon.` @AnOpenSauceDev pls gimme perms

you can just directly push to a git remote, things like this need a pull request

**Comment by Cyclopropinon** [comment]
does that mean i have to make a new fork?

**Comment by AnOpenSauceDev** [comment]
> does that mean i have to make a new fork?

not quite, you should be able to commit the same file to master on your existing fork, then PR that.

**Comment by Cyclopropinon** [comment]
ah
lemme try

**Comment by Cyclopropinon** [comment]
it worked

**Comment by AnOpenSauceDev** [comment]
Aside from what i think is a typo, this looks good to me.
Make a pull request if you want it merged now.

**Comment by Cyclopropinon** [comment]
i can do that
but ill first add a few more langs

**Comment by Cyclopropinon** [comment]
there u go:
https://github.com/AnOpenSauceDev/Methane-mod/pull/36
